### ISSUE TITLE
Is the pre-trained model trained using both the FLIC AND MPII dataset?

### ISSUE THREAD

**Issue by neherh** [anewell/pose-hg-train#44]
Is the pre-trained model trained using both the FLIC AND MPII dataset? Or is it just pre-trained on one of the datasets? In your paper, you are unclear as to whether you trained on both or separately.

**Comment by anewell** [comment]
The pretrained model is trained only on MPII. Benchmarking for the paper is done by training and evaluating on a each dataset separately.

**Comment by vicdu** [comment]
The result in the paper is 90.09, but the recurring result is 88.78, where do you think the problem occurred?

**Comment by anewell** [comment]
If you're getting 88.78 on validation then that's probably fine. The performance on the official test set (what we report in the paper) is always different, and usually a little bit higher than validation performance in my experience.

**Comment by vicdu** [comment]
@anewell thanks. I have some other stupid questions cause I‘m first submit the test set result. Did you emailed predictions to [eldar at mpi-inf.mpg.de] and get the 90.09？Is the format of predictions 1*6908 struct mat type? How long will he reply to you? thanks.
